ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -19,9 +19,8 @@ import models.CuratedRecipeForm._
 class Application(override val wsClient: WSClient, override val conf: Configuration, db: DB, val messagesApi: MessagesApi) extends Controller with AuthActions with I18nSupport with StrictLogging {
 
   def index = AuthAction { implicit request =>
-    val progressBarPercentage: Double = db.progressBarRatio() * 100 // expected to be between 0 and 100
-    val progressBarPercentageRounded = progressBarPercentage - progressBarPercentage % 0.001
-    Ok(views.html.app("Recipeasy", progressBarPercentageRounded))
+    val progressBarPercentage: Double = ( db.progressBarRatio() * 10000 ).toInt.toDouble/100 // expected to be between 0 and 100
```

Comment:
`toInt.toDouble` is probably not what you want.
I think you may have the same result with:
```scala
( db.progressBarRatio() * 10000 ) / 100.00
```